cluster/juju/layers/kubernetes-worker/tests/10-deploy

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+
+import amulet
+import requests
+import unittest
+
+
+class TestCharm(unittest.TestCase):
+    def setUp(self):
+        self.d = amulet.Deployment()
+
+        self.d.add('kubernetes-worker')
+        self.d.expose('kubernetes-worker')
+
+        self.d.setup(timeout=900)
+        self.d.sentry.wait()
+
+        self.unit = self.d.sentry['kubernetes-worker'][0]
+
+    def test_service(self):
+        # test we can access over http
+        page = requests.get('http://{}'.format(self.unit.info['public-address']))
```

Comment:
I know this is a boilerplate test, but we should improve it a little bit to make sure the services are running inside the charm. 

run('ps -ef | grep kubelet') and run('ps -ef | grep kube-proxy).  Something like that?